pkg/normalizer/service_name.go

```diff
@@ -54,7 +55,7 @@ func newServiceNameReplacer() *strings.Replacer {
 	oldnew := make([]string, 0, 2*(256-2-10-int('z'-'a'+1)))
 	for i := range mapping {
 		if mapping[i] != byte(i) {
-			oldnew = append(oldnew, string(i), string(mapping[i]))
+			oldnew = append(oldnew, fmt.Sprintf("%c", i), fmt.Sprintf("%c", mapping[i]))
```

Comment:
I think the recommendation from 1.15 release notes is to use `string(rune(i))`

Reply:
updated, thanks